### ISSUE TITLE
Extracted save copied back is not working.

### ISSUE THREAD

**Issue by whitezombie2000** [slinga-homebrew/Save-Game-Copier#7]
I extracted a Dragon Force 2 save (this game has an English patch applied), ran the python script from Save-Game-Extractor, I did this twice. I have not tried copying any other games yet.

@ubuntu3:~/Save-Game-Extractor$ python3 sgex.py mysave.bin
Save Game Extractor
(github.com/slinga-homebrew/Save-Game-Extractor)

Errors Corrected: 4
Transmitted Filename: DF2_DATA_01
Transmitted Save Size: 8636
Transmitted MD5: 883bd16501990a5c3cbef12f6f01309d
Computed MD5: 883bd16501990a5c3cbef12f6f01309d

MD5 hashes validate, save is correct.
Wrote save game DF2_DATA_01 to disk

@ubuntu3:~/Save-Game-Extractor$ python3 sgex.py df2.bin
Save Game Extractor
(github.com/slinga-homebrew/Save-Game-Extractor)

Errors Corrected: 2
Transmitted Filename: DF2_DATA_01
Transmitted Save Size: 8636
Transmitted MD5: 883bd16501990a5c3cbef12f6f01309d
Computed MD5: 883bd16501990a5c3cbef12f6f01309d

MD5 hashes validate, save is correct.
Wrote save game DF2_DATA_01 to disk

I compiled a bin+cue with the converted save DF2_DATA_01 twice and used Save-Game-Copier to copy to the internal memory. 

https://imgur.com/a/BiSKHv8

File copied to the internal memory via Save-Game-Copier.
https://imgur.com/a/jddTISd

This is what it looks like on my cartridge which is where I extracted the save using Save-Game-Extractor.

https://imgur.com/a/5yEYeKK

This what the file looks like that has been copied by Save-Game-Copier.

https://imgur.com/a/n11xYm3

I go into the game and there is no save file.

https://imgur.com/a/P1CKCqs

**Comment by whitezombie2000** [comment]
I tried Albert Odyssey and have the exact same issue. The save file has been copied to the internal RAM but I can't access it at in game continue screen.

**Comment by slinga-homebrew** [comment]
Can you rename the file to ALBERT_G._01?

**Comment by whitezombie2000** [comment]
This is the file dumped from a Saturn Gamer's Cartridge and compared with a raw save from the python script.

https://imgur.com/a/mtCTsh8

This is the file dumped from SFF and compared with a raw save from the python script.

https://imgur.com/a/k9moqat

**Comment by slinga-homebrew** [comment]
SSF adds a 16 byte header IIRC

**Comment by whitezombie2000** [comment]
I renamed the file to ALBERT_G._01 and recompiled. I copied the save back to the Saturn but it appears to be corrupt showing some garbage in the lower left of the screen. Then freezes when I load it.

**Comment by slinga-homebrew** [comment]
The md5 hashes are correct?

**Comment by slinga-homebrew** [comment]
The ones that SGC shows and the md5 from the game

**Comment by whitezombie2000** [comment]
It looks like the md5 changed after I renamed the file.

**Comment by whitezombie2000** [comment]
Wait a sec I had the wrong file. Let me try again.

**Comment by slinga-homebrew** [comment]
Verify the MD5 on your machine and then after SGC copies it to the Saturn.

**Comment by whitezombie2000** [comment]
Ok I renamed the proper game save file to ALBERT_G._01, recompiled, copied it and it worked.

**Comment by slinga-homebrew** [comment]
Woot.

**Comment by whitezombie2000** [comment]
So do all game save files need to renamed manually? Is this an issue when they are dumped?

**Comment by slinga-homebrew** [comment]
Save Game Copier is still in alpha, need to work on it more. But yeah the files still need to be 8.3.

**Comment by whitezombie2000** [comment]
Thanks for your help! Have a good night!
